doc/plugin_tutorial/dune

```diff
@@ -0,0 +1,10 @@
+; We get a super bizarre error here...
```

Comment:
Fix?

Reply:
I don't know what's going on in the first place.